### PR TITLE
WYSIWYG

### PR DESCRIPTION
**Pull request by thomasjthomasj**
This PR installs the <a href="https://github.com/daviferreira/medium-editor">Medium WYSIWYG</a> editor with the <a href="https://github.com/IonicaBizau/medium-editor-markdown">Markdown</a> plugin.

Included in the pull request:

+ WYSYWYG form field
+ Richtext content field override that uses the WYSIWYG form field instead of a text area
+ JavaScript included in the form view to handle the the WYSIWYG
+ Faux-links to switch between the WYSIWYG and a regular text area
+ An AJAX request is made if the content has been edited in the regular text area when switching back to the WYSIWYG, in order to parse the markdown to HTML

### CODE REVIEW

**Comment by thomasjthomasj** [comment]
screen shot 2015-05-14 at 11 31 24
This is annoying, @aislingbrock is there anyway to ensure this doesn't happen (by tying it to the textarea rather than the surrounding div, perhaps?) or is it something that can only be resolved by fixing the issue with the textarea default size?

File: resources/assets/js/components/medium-editor/markdown.converter.js [hunk 1]
```diff
@@ -0,0 +1,1627 @@
+"use strict";
```
Comment (kuiche) [inline resources/assets/js/components/medium-editor/markdown.converter.js]: Is this file used, I thought we went with aJax

File: resources/assets/js/components/medium-editor/me-markdown.standalone.js [hunk 2]
```diff
@@ -0,0 +1,289 @@
+(function (root) {
```
Comment (kuiche) [inline resources/assets/js/components/medium-editor/me-markdown.standalone.js]: Same here

Comment (thomasjthomasj) [inline resources/assets/js/components/medium-editor/me-markdown.standalone.js]: This one *is* used

Comment (kuiche) [inline resources/assets/js/components/medium-editor/me-markdown.standalone.js]: oh, this is the medium plugin? nm then.

**Pull request merged**